xoinvader/render.py

```diff
@@ -56,6 +60,9 @@ def render_objects(objects, screen):
                     obj.remove_obsolete(gpos)
                     continue
 
+                if image in INVISIBLE_SYMBOLS:
```

Comment:
in works perfect in python and continue too, no need to test them. When renderer will be more complicated I cover it whole by tests.